### ISSUE TITLE
Touch 

### ISSUE THREAD

**Issue by szybki8562** [ekymoz/homeRoughEditor#1]
hello plese an example how to touch move lines

**Comment by szybki8562** [comment]
```
edit engine.js

document.querySelector('#lin').addEventListener("mouseup", _MOUSEUP);
document.querySelector('#lin').addEventListener("touchend", _MOUSEUP);
document.querySelector('#lin').addEventListener("mousemove", throttle(function(event){ _MOUSEMOVE(event);},30));
document.querySelector('#lin').addEventListener("touchmove", throttle(function(event){ _MOUSEMOVE(event);},30));
document.querySelector('#lin').addEventListener("mousedown", _MOUSEDOWN, true);
document.querySelector('#lin').addEventListener("touchstart", _MOUSEDOWN, true);
```

this work

**Comment by flatsiedatsie** [comment]
I'd actually recommend not having two listeners active at the same time, and selecting either the touch option or the mouse option. I saw some weird behaviour when I had both active.

See:
https://github.com/createcandle/homeRoughEditor/blob/d991a8194bed4f1ff9b922eef0f8cec612ec21ee/index.html#L1154

and the actual if statement:

https://github.com/createcandle/homeRoughEditor/blob/d991a8194bed4f1ff9b922eef0f8cec612ec21ee/index.html#L1318

Another thing I found usefil is to quicky call `_MOUSEMOVE` once right before touch start. In my case this allowed for touch-based selection of objects. See:

https://github.com/createcandle/homeRoughEditor/blob/d991a8194bed4f1ff9b922eef0f8cec612ec21ee/index.html#L1422
